chromium_src/third_party/blink/renderer/modules/storage/dom_window_storage.cc

```diff
@@ -87,24 +96,26 @@ EphemeralStorageNamespaces* EphemeralStorageNamespaces::From(
   if (supplement)
     return supplement;
 
-  Document* document = window->GetFrame()->GetDocument();
-  auto* web_frame = WebLocalFrameImpl::FromFrame(document->GetFrame());
+  auto* web_frame = WebLocalFrameImpl::FromFrame(window->GetFrame());
   WebViewImpl* webview = web_frame->ViewImpl();
   if (!webview || !webview->Client())
     return nullptr;
-  String session_storage_id =
+  String session_storage_id = StringToSessionStorageId(
       String::FromUTF8(webview->Client()->GetSessionStorageNamespaceId()) +
-      String("/ephemeral-session-storage");
+      "/ephemeral-session-storage");
```

Comment:
I'd really like to move this concatenation `+ /ephemeral-session-storage` into `StringToSessionStorageId()`

Reply:
:+1: